FAQ.md

```diff
@@ -210,16 +210,19 @@ way, when you are searching down some path that's already longer than
 this shortest route, you can just stop and avoid wasted effort. In
 sequential land, you might model this "best result" as a shared value
 like `Rc<Cell<usize>>` (here the `usize` represents the length of best
-path found so far); in parallel land, you'd use a `Arc<AtomicUsize>`.
+path found so far); in parallel land, you'd use a `&AtomicUsize`.
+Note that `Arc` is no longer necessary because atomic types provide interior mutability.
```

Comment:
This note doesn't make sense -- the earlier `Cell` also provides interior mutability, but `Rc` and `Arc` are for shared ownership. Let's leave that change out, because it's unrelated to the race you identified.

Reply:
I was thinking it doesn't make sense to wrap an AtomicUsize in Arc, since it's redundant interior mutability, I'll revert it back so there's no change.

Reply:
The interior mutability in `Arc<T>` is *only* for its own reference counter, not the `T` value.